### ISSUE TITLE
Second day missing voice file?

### ISSUE THREAD

**Issue by Grelo** [07th-mod/himatsubushi#2]
On second day the line "Are you going to work Makino?" and the next sentence has no voice file. Does PS3 files include it or?

**Comment by ItaloKnox** [comment]
Yeah, just double checked and that's how the PS3 script goes:

```
[...]
S05/27/100500011    「…にぱ～～☆」
S05/27/100500012    「…村を観光したい若者なのです」
S05/13/101300169    「ま、…まぁね。
S05/13/101300170    この村の自然がとても貴重だと聞いたんで、…ぜひ、カメラに収めたいと思ったんだよ。
```

If we had voice files for that, it should be between 100500011  and 100500012. Maybe they were voiced in the PS2 version; do you still have the files, @enumag?

**Comment by enumag** [comment]
I do have the files but... there can't really be anything between 100500011 and 100500012. In the case I found in Tatari there was a gap in the numbers which is how I found it. This way I have no idea what file it might be.

**Comment by ItaloKnox** [comment]
So I guess the voice files doesn't come with a list like we have for the PS3 graphics? That sucks, I'll try to take a look at it later. At least my work is already done, won't hurt to at least look for it.

**Comment by enumag** [comment]
I looked into the PS3 script and it seems this line doesn't have a voice file because it did not exist in the PS3 version. Instead Rika's question "Are you going to work Makino?" and Makino's anwer "It's not exactly work, per se.  I was asked by the mayor to show around a young'un who came to the village for some sightseeing." was skipped on PS3 with this:

…と、少女はふいに私に顔を向けて、可愛らしい仕草で小首をかしげる。それを見て老人は私のことに気がついて、雛見沢を訪れた観光客だと紹介してくれた。

google translate:
... and the girl is looking at me at random and scratches his little head with a pretty gesture. When seeing it, the old man noticed me and introduced me as a tourist who visited Hina Masuzawa.

We can either leave it voice-less or replace it with the PS3 version. I think we should do the latter.

**Comment by ItaloKnox** [comment]
Maybe changing it is not too bad, doesn't seem to censor anything. I'm not really good with long sentences in japanese, but that's the best I could adapt it to:
``...So, the girl turns her face at me, tilting her head a little in a cutesy manner. After that, the elder finally notices me and introduces me as the tourist who came to sightsee Hinamizawa.``

**Comment by enumag** [comment]
Thanks. Since you're not all that confident, perhaps @DoctorDiablo could check the translation for us?

**Comment by ItaloKnox** [comment]
I don't think the translation is wrong (I didn't go for a literal translation, tho) but I think it could use a little more work. Maybe making it sound more natural, since I'm not a native speaker.

**Comment by ItaloKnox** [comment]
Asked someone who's way better at japanese than me, so that's the corrected version:
>...So, the girl turns her face at me, tilting her head a little in a cutesy manner. After seeing that, the elder introduces himself as a tourist who came to sightsee Hinamizawa.

**Comment by enumag** [comment]
Actually from the context of the dialogue I'm pretty sure your first translation is the correct one so I'll go with that.

**Comment by enumag** [comment]
Well actually both feel a little wrong. From the context it should be like this:

> ...So, the girl turns her face at me, tilting her head a little in a cutesy manner. After seeing that, the elder introduces **me** as a tourist who came to sightsee Hinamizawa.

**Comment by ItaloKnox** [comment]
Agreed, I didn't have context to show so the single excerpt might give away the wrong impression of what's happening. I think this looks fine, I would go ahead with it.

**Comment by enumag** [comment]
Fixed in #18.
